test/project_types/script/layers/infrastructure/script_repository_test.rb

```diff
@@ -67,48 +68,54 @@
       end
 
       it "should raise ScriptNotFoundError when script source file does not exist" do
-        FileUtils.mkdir_p(script_source_base)
+        context.mkdir_p(script_source_base)
         e = assert_raises(Script::Layers::Domain::Errors::ScriptNotFoundError) { subject }
         assert_equal script_source_file, e.script_name
       end
     end
   end
 
   describe ".with_temp_build_context" do
-    let(:script_file) { "#{extension_point.type}.#{language}" }
+    let(:script_file) { "script.#{language}" }
     let(:helper_file) { "helper.#{language}" }
 
     before do
-      FileUtils.mkdir_p(script_source_base)
+      # byebug
+      context.mkdir_p(script_source_base)
       Dir.chdir(script_source_base)
-      File.write(script_file, "//run code")
+      context.root = script_source_base
+      context.write(script_file, "//run code")
     end
 
     it "should go to a tempdir with all its files" do
-      File.write(helper_file, "//helper code")
-      FileUtils.mkdir_p("other_dir")
+      # ctx = TestHelpers::FakeContext.new
+      context.write(helper_file, "//helper code")
+      context.mkdir_p("other_dir")
 
-      script_repository.with_temp_build_context do
+      script_repository.with_temp_build_context(context) do
         refute_equal script_source_base, Dir.pwd
+        # pry
         assert File.exist?(script_file)
         assert File.exist?(helper_file)
       end
     end
 
     it "should create temp directory in the script root" do
+      # byebug
```

Comment:
Oops - I think you forgot to delete all these comments in this file from when we debugged 😅